src/Configuration/Dotenv/Scheme/Types.hs

```diff
@@ -21,33 +23,17 @@ instance FromJSON EnvType where
   parseJSON (String x) = fail ("Don't know how to parse that kind of type: " ++ show x)
   parseJSON x = fail ("Not an object: " ++ show x)
 
-newtype Config = Config [EnvConf] deriving (Show, Eq)
-
-instance FromJSON Config where
-  parseJSON = fmap Config . parseJSONList
-
-data EnvConf =
-  EnvConf
-    { envType  :: EnvType
-    , envs     :: [Env]
-    } deriving (Show, Eq)
-
-instance FromJSON EnvConf where
-  parseJSON (Object m) =
-    EnvConf
-      <$> m .: "type"
-      <*> m .: "envs"
-  parseJSON x = fail ("Not an object: " ++ show x)
-
 data Env =
   Env
     { envName  :: String
+    , envType  :: EnvType
     , required :: Bool
     } deriving (Show, Eq)
 
 instance FromJSON Env where
   parseJSON (Object m) =
     Env
       <$> m .: "name"
-      <*> m .: "required"
+      <*> m .: "type"
+      <*> (fromMaybe False <$> m .:? "required")
```

Comment:
@CristhianMotoche what about:

```hs
m .:? "required" .!= False
```

There is already a function provided by [aeson](http://hackage.haskell.org/package/aeson-1.2.3.0/docs/Data-Aeson.html#v:.-33--61-)